docs/content/maintenance/metrics.md

```diff
@@ -0,0 +1,399 @@
+        </tr>
+        <tr>
+            <td>lastDeltaRecordsCommitCompacted</td>
+            <td>Table</td>
+            <td>Gauge</td>
+            <td>Delta records count in last commit with COMPACT commit kind.</td>
+        </tr>
+        <tr>
+            <td>lastChangelogRecordsCommitCompacted</td>
+            <td>Table</td>
+            <td>Gauge</td>
+            <td>Changelog records count in last commit with COMPACT commit kind.</td>
+        </tr>
+        <tr>
+            <td>lastPartitionsWritten</td>
+            <td>Table</td>
+            <td>Gauge</td>
+            <td>Number of partitions written in the last commit.</td>
+        </tr>
+        <tr>
+            <td>lastBucketsWritten</td>
+            <td>Table</td>
+            <td>Gauge</td>
+            <td>Number of buckets written in the last commit.</td>
+        </tr>
+    </tbody>
+</table>
+
+### Compaction Metrics
+
+<table class="table table-bordered">
+    <thead>
+    <tr>
+      <th class="text-left" style="width: 225pt">Metrics Name</th>
+      <th class="text-left" style="width: 65pt">Level</th>
+      <th class="text-left" style="width: 70pt">Type</th>
+      <th class="text-left" style="width: 300pt">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>lastCompactionDuration</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>The time it took to complete the last compaction.</td>
+        </tr>
+        <tr>
+            <td>compactionDuration</td>
+            <td>Bucket</td>
+            <td>Histogram</td>
+            <td>Distributions of the time taken by the last few compaction.</td>
+        </tr>
+        <tr>
+            <td>lastTableFilesCompactedBefore</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Number of deleted files in the last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastTableFilesCompactedAfter</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Number of added files in the last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastChangelogFilesCompacted</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Number of changelog files compacted in last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastRewriteInputFileSize</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Size of deleted files in the last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastRewriteOutputFileSize</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Size of added files in the last compaction.</td>
+        </tr>
+        <tr>
+            <td>lastRewriteChangelogFileSize</td>
+            <td>Bucket</td>
+            <td>Gauge</td>
+            <td>Size of changelog files compacted in last compaction.</td>
+        </tr>
+    </tbody>
+</table>
+
+# Bridging To Flink
```

Comment:
```suggestion
## Bridging To Flink
```

Level 1 headers are only used for page titles.

Reply:
@tsreaper Got it, thanks for reminding~